sanic/exceptions.py

```diff
@@ -154,87 +155,3 @@ def __init__(self, message, content_range):
 
 class InvalidRangeType(ContentRangeError):
     pass
-  
-  
-class Handler:
```

Comment:
The name also changed as well but I don't think too many people were subclassing it but we can add back a soft-link if people need it for legacy stuff